D:SWARM-Use-Case-at-SLUB-Dresden.md

```diff
@@ -0,0 +1,59 @@
+At [SLUB Dresden](http://www.slub-dresden.de), the *Streaming version* of d:swarm is used for normalising data retrieved from various sources before feeding it to a [Solr](http://lucene.apache.org/solr/) server. Clients such as the new [SLUB Katalog (beta)](http://katalogbeta.slub-dresden.de/) can then request data from Solr and present it to library users.
+
+[[img/use-case-slub.png]]
+
+## Data Sources and Formats
+
+The following data sources are currently processed with d:swarm at SLUB Dresden:
+
+* [deutsche FOTOTHEK](http://www.deutschefotothek.de)
+  * OAI-PMH-XML (containing elements from the [DC-Elements](http://purl.org/dc/elements/1.1/) and [Europeana](http://europeana.eu/terms) namespace)
+* Ebook Library, E-Book packages from Wiley, deGruyter (EBL, WILEY, DEG)
+  * [OAI-PMH-MARCXML](http://www.loc.gov/standards/marcxml/)
+* [Gemeinsamer Bibliotheksverbund](http://www.gbv.de/) (GBV)
+  * MARCXML (currently wrapped as OAI-PMH-MARCXML as well, to avoid too many different source data formats)
+
+The target schema we map to, is a schema describing the format that [Finc](https://finc.info/de/) is using for feeding the Solr index. To be able to merge the data with other sources provided via Finc, we export the data as XML compliant to this schema. Below, you find an example of a source format (OAI-PMH + DC-elements and Europeana) on the left and the export format (Finc Solr schema compliant XML) on the right hand side.
```

Comment:
"To be able to merge the data with other sources provided via Finc, we export the data as XML compliant to this schema." -> "We export the transformation result of our mapped data as XML (to be able) to import it into the Solr search index (where it will be merged with data from other sources)."